yourgov/components/LinkComponent.tsx

```diff
@@ -14,7 +14,10 @@ export const LinkComponent = forwardRef<HTMLAnchorElement, LinkComponentProps>(
 		// Use an `a` tag when linking externally
 		// Regex finds links starting with: `http://` | `https://` | `//`
 		const hrefAsString = typeof href === 'string' ? href : href?.pathname;
-		if (hrefAsString && /^(https?:\/\/|\/\/)/i.test(hrefAsString)) {
+		if (
+			hrefAsString &&
+			(/^(https?:\/\/|\/\/)/i.test(hrefAsString) || hrefAsString?.includes('#'))
```

Comment:
What was this change for?

Reply:
This allows linking from the error alert. `Next/Link` doesn't do in page navigation with `#`, so it needs to be a basic `<a>`.